instrumentation/kamon-annotation/src/main/java/kamon/annotation/instrumentation/cache/AnnotationCache.java

```diff
@@ -24,29 +26,26 @@
 import kamon.metric.*;
 import kamon.tag.TagSet;
 import kamon.trace.SpanBuilder;
-import kanela.agent.libs.net.jodah.expiringmap.ExpirationListener;
-import kanela.agent.libs.net.jodah.expiringmap.ExpirationPolicy;
-import kanela.agent.libs.net.jodah.expiringmap.ExpiringMap;
 import kanela.agent.util.log.Logger;
 
 import java.lang.reflect.Method;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Objects;
+import java.util.concurrent.ConcurrentMap;
 import java.util.concurrent.TimeUnit;
 
 public final class AnnotationCache {
 
-  private static Map<MetricKey, Object> metrics = buildCache();
+  private static ConcurrentMap<MetricKey, Object> metrics = buildCache();
```

Comment:
why not use the interface `Map<MetricKey, Object>` instead of the implementation `ConcurrentMap<MetricKey, Object>` in the variable declaration?

Reply:
Must have slipped through, nice catch!